requirements.txt

```diff
@@ -3,4 +3,5 @@ Django==4.1.10
 djangorestframework==3.13.1
 pytz==2022.1
 sqlparse==0.4.4
-# mysqlclient==1.4.3
\ No newline at end of file
+# mysqlclient==1.4.3
+mysql-connector-python==8.2.0
```

Comment:
Add Eof(End-of-file), need to add empty line at the end
![image](https://github.com/mate-academy/devops_todolist_docker_core_task_2_volumes/assets/87720583/81c1421c-41bc-4b0a-941f-1294da8357dd)